deploy/esignet/install.sh

```
@@ -97,28 +259,254 @@ function installing_esignet() {
+  } > "$plugin_env_file"
+
+  plugin_option="--set pluginNameEnv=$plugin_name -f $plugin_env_file"
 
   echo Installing esignet
-  helm -n $NS install esignet mosip/esignet --version $CHART_VERSION \
-  $ESIGNET_HELM_ARGS \
-  --set image.repository=mosipdev/esignet --set image.tag=develop \
-  $ENABLE_INSECURE $plugin_option \
-  --set metrics.serviceMonitor.enabled=$servicemonitorflag -f values.yaml --wait
+  helm -n $NS install $ESIGNET_SERVICE_NAME /home/techno-467/IdeaProjects/esignet/helm/esignet --version $CHART_VERSION  \
+    $ENABLE_INSECURE $plugin_option \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)